ftl/php/builder.py

```diff
@@ -33,27 +34,38 @@ def __init__(self, ctx, args, cache_version_str):
         super(PHP, self).__init__(ctx, _PHP_NAMESPACE, args, cache_version_str,
                                   [_COMPOSER_LOCK, _COMPOSER_JSON])
 
+    def _parse_composer_pkgs(self):
+        descriptor_contents = ftl_util.descriptor_parser(
+            self._descriptor_files, self._ctx)
+        composer_json = json.loads(descriptor_contents)
+        pkgs = []
+        for k, v in composer_json['require'].iteritems():
+            pkgs.append(k + ' ' + v)
```

Comment:
This feels a bit hacky - could we keep this as a tuple or some other structure? It looks like you split this back apart by space later on anyway.

Reply:
I changed this to be a tuple